README.md

```diff
@@ -127,14 +127,14 @@ https://dbdiagram.io/d/60769bb6b6aeb3052d90059f - Use https://dbdiagram.io/
 Generally there are 2 types of components: _presentational_ and _container_ components. Please review the differences below:
 
 |                       | Presentational components                           | Container components                                                                                                                                                      |
-| --------------------- | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| --------------------- | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
```

Comment:
Please exclude the README.md file from the PR

Reply:
Exceptionally we will accept this change on the README file